awx/main/tests/settings_for_test.py

```diff
@@ -7,6 +7,9 @@
 # Some things make decisions based on settings.SETTINGS_MODULE, so this is done for that
 SETTINGS_MODULE = 'awx.settings.development'
 
+# Turn off task submission, because sqlite3 does not have pg_notify
+DISPATCHER_MOCK_PUBLISH = True
```

Comment:
do the live tests run using sqlite3? or does the setting here only affect unit tests?

Reply:
The live tests use postgres.

Correct, the setting here only affects tests using the sqlite3 database (you could call "unit" tests, but that lacks clarity due to abuse of the term). Using this new contract, the `is_testing` criteria is avoided, which would result in true for any pytest tests.